packages/app/components/Nav/NavBar.tsx

```diff
@@ -8,6 +8,7 @@ import Link from "next/link";
 import { API } from "@template/api-client";
 import useSWR from "swr";
 import { useProfile } from "../../hooks/useProfile";
+import Settings from "./Settings/Profile";
```

Comment:
Is there a reason we are importing this? It seems like it isn't being used in this file

Reply:
Good catch! I'm removing all unused imports